ref/general/about-open-liberty.adoc

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2020 IBM Corporation and others.
+// Licensed under Creative Commons Attribution-NoDerivatives
+// 4.0 International (CC BY-ND 4.0)
+//   https://creativecommons.org/licenses/by-nd/4.0/
+//
+// Contributors:
+//     IBM Corporation
+//
+:page-description: Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+:page-layout: general-reference
+:seo-title: About Open Liberty
+:seo-description: Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+:page-layout: general-reference
+:page-type: general
+= About Open Liberty
+
+Open Liberty is a lightweight Java runtime for building cloud-native applications and microservices.
+
+With Open Liberty, you can add and remove modular features from the latest versions of Eclipse MicroProfile and Jakarta EE.
+This modular structure simplifies application development, so you run only the features that your application needs.
+Furthermore, thanks to zero-migration architecture, you can upgrade to the latest version of Open Liberty with minimal impact to your current applications and configurations.
+Open Liberty is compatible with MicroProfile and with the Jakarta EE Full Platform and Web Profile specifications.
+For updates about Open Liberty features and capabilities, see the link:https://openliberty.io/blog/[Open Liberty blog] and https://openliberty.io/docs/[Open Liberty docs].
+
+Open Liberty provides a diverse library of interactive and static developer guides that are designed to give you firsthand experience with the configuration and features that support cloud-native development.
+The guides cover a wide range of development and production scenarios and can be adapted as reference templates for developing your own applications. link:/guides/[Browse the guides] and learn how to build, package, and deploy applications and microservices with Open Liberty.
+
+
+== Develop cloud-native Java applications and microservices
+
+Open liberty is designed to simplify and accelerate application development. Open Liberty offers a low-memory footprint and some of link:/blog/2019/10/30/faster-startup-open-liberty.html[the fastest start-up times of any lightweight Java server]: one second or less for simple configurations on processors that are used in current laptops and servers.
+Faster start-up times enable faster development and test cycles.
+
+=== Mix and match features
+Open Liberty features are pluggable, so that your existing features are supported in new product versions. You can add, remove, and update features as needed, with minimal impact to your application. When the feature configuration is changed, Open Liberty reevaluates the code by starting and stopping parts of the runtime as necessary, without having to restart the server. For a full list of available features and their configuration elements, see link:/docs/ref/feature/#featureOverview.html[Server features].
```

Comment:
Change, "with minimal impact to your application." to "with minimal impact on your application."